nyxx.interactions/lib/src/ExtendNyxx.dart

```diff
@@ -0,0 +1,3 @@
+part of nyxx_interactions;
+
+extension on Nyxx {}
```

Comment:
Is that leftof or we have plans for this?

Reply:
There were plans however they have since changed, file will be deleted in next push.